lib/formatters/stylish.js

```diff
@@ -73,14 +77,22 @@ module.exports = function(results) {
         ).split("\n").map(el => el.replace(/(\d+)\s+(\d+)/, (m, p1, p2) => chalk.dim(`${p1}:${p2}`))).join("\n")}\n\n`;
     });
 
-    const total = errors + warnings;
+    const total = errorCount + warningCount;
 
     if (total > 0) {
         output += chalk[summaryColor].bold([
             "\u2716 ", total, pluralize(" problem", total),
-            " (", errors, pluralize(" error", errors), ", ",
-            warnings, pluralize(" warning", warnings), ")\n"
+            " (", errorCount, pluralize(" error", errorCount), ", ",
+            warningCount, pluralize(" warning", warningCount), ")\n"
         ].join(""));
+
+        if (fixableErrorCount > 0 || fixableWarningCount > 0) {
+            output += chalk[summaryColor].bold([
+                "  ", fixableErrorCount, pluralize(" error", fixableErrorCount), ", ",
+                fixableWarningCount, pluralize(" warning", fixableWarningCount),
+                " can potentially be fixed by running eslint with the `--fix` option.\n"
```

Comment:
I'm open for comments in the message too. I think the differentiation between errors and warnings may useful because not everyone wants to automatically fix warnings.

What about:
> 2 problems (1 error, 1 warning)
> 1 error, 1 warning potentially fixable with the `--fix` option